tests/end2end/features/caret.feature

```diff
@@ -104,3 +104,28 @@ Feature: Caret mode
         And I run :fake-key <tab>
         And I run :follow-selected --tab
         Then data/hello.txt should be loaded
+        
+    # :reverse-selection
```

Comment:
These tests should be written in Python and in `tests/unit/browser/test_caret.py`, not here. The tests in here are end2end tests which are rather slow and brittle, and qutebrowser is trying to move away from them. Here, I don't see a reason why those tests would need to be here.

Reply:
So, should I fix up the feature tests that I wrote here, or get rid of them altogether and write tests in `tests/unit/browser/test_caret.py`?

Reply:
I just noticed that you said which file to write the tests in earlier. Sorry bout that.